feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch
```

Comment:
it's really good that you have decide to add this line of code

Reply:
Thanks for this comment
bolt